### PR TITLE
Add `Task.labeledTimeout`

### PR DESCRIPTION
**Pull request by leifwickland**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ import scalaz.Catchable
 import scalaz.syntax.catchable._
 import scalaz.effect.IO
 import rubiz.syntax.catchable._
+import scala.concurrent.duration._
```
Comment (skeet70) [inline README.md]: Move this to the `Task` set of imports.

File: src/main/tut/README.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ import scalaz.Catchable
 import scalaz.syntax.catchable._
 import scalaz.effect.IO
 import rubiz.syntax.catchable._
+import scala.concurrent.duration._
```
Comment (skeet70) [inline src/main/tut/README.md]: Sorry move it in this place, not the generated one :D

**Review by skeet70 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (leifwickland) [reply README.md]: Good catch. Thanks.

File: src/main/tut/README.md (hunk 2 above)
Comment (leifwickland) [reply src/main/tut/README.md]: Done.

**Review by skeet70 — APPROVED** [review]

**Pull request merged**